packages/aws/data_stream/lambda/fields/fields.yml

```
@@ -78,6 +94,15 @@
               type: long
               metric_type: gauge
               description: The number of times your function code is executed on standard concurrency when all provisioned concurrency is in use.
+            - name: AsyncEventsReceived.sum
+              type: long
+              metric_type: gauge
+              description: The total number of events that Lambda successfully queues for processing.
+            - name: AsyncEventAge.avg
+              type: long
+              unit: ms
+              metric_type: gauge
+              description: The average time between whem Lambda successfully queues the event and when the function is invoked.
```

```suggestion
              description: The average age of the asynchronous events in the function event queue.
```

Two issues here:
- Spelling of "when"
- The sentence does not make sense. Avg time between? If you read it, it is very hard to understand. So I have suggested my edit.

---

Updated as suggested.